### PR TITLE
Individual restaurant screen

### PR DESCRIPTION
**Pull request by EricNatelson**

### CODE REVIEW

File: react-ui/src/components/Restaurant-info/Restaurant-info.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React, { useCallback, useEffect, useState, Fragment } from 'react';
+
+function RestaurantInfo(props) {
+    return <h1>Render this</h1>
```
[the remark this answers is not in the document]
Comment (benhurley) [reply react-ui/src/components/Restaurant-info/Restaurant-info.js:4]: hound being an IDIOT

File: react-ui/src/components/Home-page/Home-page.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { useCallback, useEffect, useState, Fragment } from 'react';
+
+
+export default function HomePage(props) {
+    return (
+        <div className="App">
```
[the remark this answers is not in the document]
Comment (benhurley) [reply react-ui/src/components/Home-page/Home-page.js:6]: No

File: react-ui/src/App.js [hunk 3]
```diff
@@ -1,33 +1,20 @@
...
-              <input type="text" name="playlist" placeholder="Music Playlist" required /> <br />
-              <input type="text" name="lighting" placeholder="Lighting" required /> <br />
-              <input type="text" name="scent" placeholder="Aromas" required /> <br />
-              <input type="submit" value="Submit" />
-              </form>
-          </Fragment>
-        }
-      </header>
-    </div>
+    <Router>
```
[the remark this answers is not in the document]
Comment (benhurley) [reply react-ui/src/App.js:14]: Wrong

File: react-ui/src/components/Home-page/Home-page.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (EricNatelson) [reply react-ui/src/components/Home-page/Home-page.js:6]: smh Hound

**Pull request merged**